web-console/4.0-designs/storage/storage.md

```diff
@@ -29,3 +29,58 @@
 ![form for adding and creating storage](img/create-storage-form.png)
 - Selecting `Create Storage` exposes fields for Storage Class, Name, Access Mode, Size, and the label selector checkbox.
 - Only access modes that are compatible should be available to select in this form.
+
+## Set Default Storage Class
+- Users with appropriate privileges can set a Default Storage Class.
+- THERE ARE TWO WAY USER CAN SET A DEFAULT STORAGE CLASS:
+
+#### OPTION-1: Kebab menu option:
+
+1. Login to OpenShift > Admin > Storage (menu) > Storage classes (sub-menu)
+2. User can see list of ‘Storage classes” [list view]
+3. There will be a label that indicates 'default storage class'
+4. User chooses a storage class row that would be changed as a ‘default storage class’
+5. Click on the kebab menu > select “Edit Default Storage Class” as shown in the screen below:
+
+![Set a Default Storage Class using kebab menu](img/stroage-opt1-start.png)
```

Comment:
I'm still wondering about being able to mark a storage class as default directly from the list view. The flow feels odd if I'm choosing from a specific storage class then I am in a modal and have to choose that storage class again. Is there any reason we can't have in the kebab the ability to mark that specific SC as default?

Reply:
I explored the same. 
What would be the kebab menu option for the "current default menu item"?

Reply:
For the Storage class that is already set as the default it could be the same text "Set as default storage class" but it would be disabled.